app/kubernetes-ingress-controller/customize-image.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Customize Images"
+
+description: |
+  Customize the image name used for {{ site.kic_product_name }} and {{ site.base_gateway }}
+
+content_type: reference
+layout: reference
+
+products:
+  - kic
+
+works_on:
+  - on-prem
+  - konnect
+---
+
+
+{{ site.kic_product_name }} supports both {{ site.base_gateway }} OSS and Enterprise. If you install {{ site.kic_product_name }} using the default Helm charts, {{ site.base_gateway }} OSS will be installed.
```

Comment:
Is this fine to say still, or should we not be mentioning OSS?

Reply:
I checked with Jessup. Let's leave this for now